code/modules/jobs/job_types/khan.dm

```diff
@@ -25,6 +25,7 @@
 	box_two = /obj/item/storage/survivalkit/medical/raider
 	backpack_contents = list(
 		/obj/item/storage/bag/money/small/khan = 1
+		/obj/item/melee/unarmed/brass/spiked = 1,
```

Comment:
Instead of the putting it into that, put it under glove slot like.
gloves = /obj/item/melee/unarmed/brass/spiked